games-puzzle/quadra/quadra-1.3.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools desktop
+
+DESCRIPTION="Tetris clone with multiplayer support"
+HOMEPAGE="https://github.com/quadra-game/quadra"
+SRC_URI="https://github.com/${PN}-game/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-libs/boost
+	media-libs/libpng:0=
+	media-libs/libsdl2[X,sound,video]
+	sys-libs/zlib"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	default
+	eautoreconf
+	sed -i \
+		-e "/^datagamesdir/s|\/games|\/${PN}|" \
+		Makefile.in || die
+}
+
+src_install() {
+	dobin ${PN}
+
+	insinto /usr/share/${PN}
+	doins ${PN}.res
+
+	doicon images/${PN}.xpm
+	make_desktop_entry ${PN} ${PN^}
+
+	einstalldocs
+}
```

Comment:
You can replace this entire src_install by just "default + make_desktop_entry", default will even install the .xpm icon, i.e. the makefile's install target works just fine.

Reply:
Nice, didn't expected that - makes it much cleaner.